bcda/cclf/parser_test.go

```diff
@@ -293,3 +293,30 @@ func TestGetCCLFMetadata(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckIfAttributionCSVFile(t *testing.T) {
+	tests := []struct {
+		name      string
+		path      string
+		testIsCSV bool
+	}{
+		{name: "Is an Attribution CSV File path", path: "P.PCPB.M2014.D00302.T2420001", testIsCSV: true},
+		{name: "Is not an Attribution CSV File path (incorrect first)", path: "T.PCPB.M2014.D00302.T2420001", testIsCSV: false},
+		{name: "Is not an Attribution CSV File path (incorrect second)", path: "P.BFD.N2014.D00302.T2420001", testIsCSV: false},
+		{name: "Is not an Attribution CSV File path (incorrect third)", path: "P.PCPB.M2014.D00302.T2420001", testIsCSV: false},
+		{name: "Is not an Attribution CSV File path (incorrect fourth)", path: "P.PCPB.M2014.D003022.T2420001", testIsCSV: false},
+		{name: "Is not an Attribution CSV File path (incorrect fifth)", path: "P.PCPB.M2014.D00302.T24200011", testIsCSV: false},
+		{name: "Is not an Attribution CSV File path (CCLF file)", path: "T.BCD.A0001.ZCY18.D181121.T1000000", testIsCSV: false},
```

Comment:
We should add an opt-out filename as well.  I am fairly sure we are using the same s3 bucket for attribution files as well as opt-out files.

If we have tests that do similar things for the CCLF file and the opt out file, we should add a CSV file example to them as well to make sure they dont pick up this filename schema.  Might be a bit overkill but I think is worth the effort.

Reply:
Updated to add an opt-out file, but couldn't find where we did tests for opt-out file names.